custom-matchers/test-asset-custom-matcher.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
not sure we need those if there is only 1 test file

Reply:
i mean scripts for test

Reply:
agreee, delete

Reply:
don`t agree, later we add more matcher(bundles-matcher for example), I think we need separate test for every matcher